src/content/docs/infrastructure/amazon-integrations/get-started/introduction-aws-integrations.mdx

```diff
@@ -121,14 +51,10 @@ This section directs you to relevant AWS docs that explain how our integrations
 * When you enable AWS Config to enrich CloudWatch metrics with resource metadata, you're charged based on the number of configuration items recorded. See the [AWS Config pricing page](https://aws.amazon.com/config/pricing/) for details.
 * [AWS Resource Groups Tagging API](https://docs.aws.amazon.com/resourcegroupstagging/latest/APIReference/overview.html) enriches metrics with custom tags defined in the associated AWS resources. This affects both polling and metric streams.
 
-New Relic uses the Amazon CloudWatch API to obtain metrics from your AWS services when you use the API Polling integration instead of CloudWatch Metric Streams. Some situations may increase the number of calls to the CloudWatch API to exceed the <DNT>**1 million free**</DNT> limits granted by AWS and [increase your CloudWatch bill](/docs/infrastructure/amazon-integrations/troubleshooting/cloudwatch-billing-increase). This may happen when you: 
-
-* Enable additional integrations through API Polling
-* Add AWS resources to those integrations
-* Scale those integrations across more regions.
+New Relic uses the Amazon CloudWatch API to capture data about AWS services when you use the API Polling integration instead of CloudWatch Metric Streams. Some situations may increase the number of calls to the CloudWatch API to exceed the <DNT>**1 million free**</DNT> limits granted by AWS and [increase your CloudWatch bill](/docs/infrastructure/amazon-integrations/troubleshooting/cloudwatch-billing-increase). This may happen when you enable additional integrations through API Polling, add AWS resources to those integrations, or scale those integrations across more regions.
 
 AWS offers enhanced monitoring for some of their services that allows for more metrics at a higher frequency. You can check out [RDS enhanced monitoring costs](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_Monitoring.OS.overview.html#USER_Monitoring.OS.cost) for more information.
 
 ## Region availability [#region-availability]
 
-Most AWS services offer regional endpoints to reduce data latency between cloud resources and applications. New Relic can obtain monitoring data from services and endpoints that are located in all [AWS regions](http://docs.aws.amazon.com/general/latest/gr/rande.html), except China.
+Most AWS services offer regional endpoints to reduce data latency between cloud resources and applications. New Relic can obtain monitoring data from services and endpoints that are located in all [AWS regions](http://docs.aws.amazon.com/general/latest/gr/rande.html), except China.
```

Comment:
are located should probably be "from all AWS regions except China." I'm not sure the last comma is necessary.